include/usgscsm/UsgsAstroFrameSensorModel.h

```diff
@@ -14,6 +14,8 @@
 #include "spdlog/sinks/basic_file_sink.h"
 
 #include <json/json.hpp>
+
+#define MESSAGE_LOG(logger, ...) if (logger) { logger->info(__VA_ARGS__); }
```

Comment:
Move this define into the source file. If it's in the header it is available elsewhere and has much higher chance of colliding with another define from some other library.